source/extensions/transport_sockets/tls/downstream_context_config_factory.h

```diff
@@ -0,0 +1,28 @@
+#include "envoy/registry/registry.h"
+#include "envoy/ssl/context_config.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace TransportSockets {
+namespace Tls {
+
+/**
+ * A factory to create server side Tls context config from a protobuf.
```

Comment:
nit: Tls/TLS here and elsewhere in comments.

Reply:
done